subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/DefaultFileWatcherRegistry.java

```diff
@@ -86,15 +87,20 @@ public void handleChangeEvent(FileWatchEvent.ChangeType type, String absolutePat
 
                             @Override
                             public void handleUnknownEvent(String absolutePath) {
+                                LOGGER.error("Received unknown event for {}", absolutePath);
```

Comment:
`LOGGER.WARN` should be good enough here. I don't know if it really matters, though.

Reply:
We've used `.error()` for the actual error event. I think it's important to be consistent, and we are talking about an error here I think, though `warn()` is fine with me if you feel strongly about it.

Reply:
For me this doesn't seem like an error, though I also don't feel strongly about moving it to warn.